src/middleware/devtools.ts

```diff
@@ -147,13 +148,37 @@ type DevtoolsImpl = <T>(
 
 export type NamedSet<T> = WithDevtools<StoreApi<T>>['setState']
 
+type ConnectResponse = ReturnType<
+  NonNullable<Window['__REDUX_DEVTOOLS_EXTENSION__']>['connect']
+>
+const connections: Record<string, ConnectResponse> = {}
```

Comment:
My suggestion for refactor is to use `Map` and deal `undefined` as a normal case.

Reply:
@dai-shi 
Ok I will use thé Map

Regarding namings, I thaught of 
storeName
slice
prefix

Reply:
`name` is to distinguish devtools instances.
`store` is to distinguish states and actions in a single instance.

I want to make that clear with naming. `prefix` might be better, but it's not a prefix. Another idea is `subName`, but not super comfortable. In a sense, it's a key. `storeKey`. Hm, then `store` would be better... ok, let's leave it for now.